omnigan/losses.py

```diff
@@ -483,6 +496,11 @@ def __call__(self, prediction, target, discriminator):
 
 
 def multiDiscriminatorAdapter(d_out, opts):
+    """
+    Because the OmniDiscriminator does not directly return a tensor (but a list of tensor).
```

Comment:
Describe the expected type of inputs and there nature. Yous should use vscode extension autodocstring to help you do that (and type hints :) )

type hints are almost like type comments: 

```
def affine_square(a: float, b: Optional[float] = 0) -> float:
    return a ** 2 + b
```

https://kishstats.com/python/2019/01/07/python-type-hinting.html